build.rs

```diff
@@ -2,17 +2,75 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+use std::env::{self, VarError};
+use std::path::{Path, PathBuf};
+
+/// Environment variable that can be set to point to the directory containing the `va.h`, `va_drm.h` and `va_drmcommon.h`
+/// files to use to generate the bindings.
+const CROS_LIBVA_H_PATH_ENV: &str = "CROS_LIBVA_H_PATH";
+
+/// Default header file to parse if the `CROS_LIBVA_H_PATH` environment variable is not set.
+const DEFAULT_CROS_LIBVA_H_PATH: &str = "/usr/include";
+
+/// Wrapper file to use as input of bindgen.
+const WRAPPER_PATH: &str = "libva-wrapper.h";
+
+/// Allow list
+const ALLOW_LIST_TYPE : &str = ".*ExternalBuffers.*|.*PRIME.*|.*MPEG2.*|.*VP8.*|.*VP9.*|.*H264.*|.*HEVC.*|VACodedBufferSegment|.*AV1.*|VAEncMisc.*|VASurfaceDecodeMBErrors|VADecodeErrorType";
+
 fn main() {
     // Do not require dependencies when generating docs.
     if std::env::var("CARGO_DOC").is_ok() || std::env::var("DOCS_RS").is_ok() {
         return;
     }
 
-    match pkg_config::probe_library("libva") {
-        Ok(_) => (),
-        Err(e) => panic!("libva not found: {}", e),
-    };
+    let va_h_path = env::var(CROS_LIBVA_H_PATH_ENV)
+        .or_else(|e| {
+            if let VarError::NotPresent = e {
+                Ok(DEFAULT_CROS_LIBVA_H_PATH.to_string())
+            } else {
+                Err(e)
+            }
+        })
+        .expect("invalid `CROS_LIBVA_H_PATH` environment variable");
```

Comment:
I would much prefer if search for libva header would be handled implicitly by the `clang`/`gcc` rather then explicit paths. On a rare occasion if one would want to compile it on windows it would fail.